packages/rich-text/src/plugins/shared/MarkPlugin.js

```diff
@@ -1,17 +1,16 @@
 import isHotkey from 'is-hotkey';
 import markDecorator from './MarkDecorator';
 import { haveMarks } from './UtilHave';
-import { css, cx } from 'emotion';
+import { css } from 'emotion';
 import tokens from '@contentful/forma-36-tokens';
 
 const styles = {
+  // TODO: Ensure a bold mark on a headline makes the heading appear even
+  //  bolder. Consider to make headings non-bold by default.
```

Comment:
This would necessitate a chat with @m10l or @joaoramos , no?

There may also be a ticket for it somewhere waaay in the vaults - I remember us talking about this during the initial implementation of the rich text editor.

Reply:
Yeah, that's why I am writing "consider" ;)
We might have decided against it in the past. Personally I still find headlines way too bold in CF RT, all the way down to h5 or h5 where bold might make sense since the text is just getting too small.